moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -49,48 +49,56 @@
 #include <fstream>
 #include <streambuf>
 #include <algorithm>
+#include <chrono>
+
+rclcpp::Logger LOGGER = rclcpp::get_logger("moveit").get_child("rdf_loader");
 
 rdf_loader::RDFLoader::RDFLoader(const std::string& robot_description)
 {
   moveit::tools::Profiler::ScopedStart prof_start;
   moveit::tools::Profiler::ScopedBlock prof_block("RDFLoader(robot_description)");
 
-  ros::WallTime start = ros::WallTime::now();
-  ros::NodeHandle nh("~");
+  auto start = std::chrono::system_clock::now();
+  // TODO (anasarrak): Add the correct node_handle name
+  auto node = rclcpp::Node::make_shared("/");
```

Comment:
This is a big TODO. What is our strategy for Nodes in MoveIt2? Should we always check that rclcpp has been initialized first? Seems like something we should do with intention.

Reply:
Suggestion: change the api for rdf_loader so that the user must pass in a node.